pkg/pgmodel/querier/querier.go

```diff
@@ -263,8 +273,9 @@ func (q *pgxQuerier) queryMultipleMetrics(filter metricTimeRangeFilter, cases []
 			rows.Close()
 			return nil, nil, err
 		}
-		// Append all rows into results.
-		results, err = appendTsRows(results, rows, nil)
+		// Append all rows into results. Schema is ignored since we only return
+		// default schema for multi-metric queries.
```

Comment:
how is this guaranteed in the code, I don't see that? Also where is this property tested?

Reply:
The SQL query uses series table to fetch the metric IDs which means it will only fetch non-view metric IDs.

While there are no specific tests, SQL query integration tests confirm this behavior.